File: Programming Basics/Java/Conditional Statements/Shopping.java
package com.company;

import java.util.Scanner;

public class Shopping {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double budget = Double.parseDouble(scanner.nextLine());
        int videoCards = Integer.parseInt(scanner.nextLine());
        int processors = Integer.parseInt(scanner.nextLine());
        int ram = Integer.parseInt(scanner.nextLine());
        double videoCardsPrice = videoCards * 250;
        double processorsPrice = 0.35 * videoCardsPrice * processors;
        double ramPrice = 0.10 * videoCardsPrice * ram;
        double totall = videoCardsPrice + processorsPrice + ramPrice;
        if(videoCards > processors) {
            totall *= 0.85;
        }
        if(budget >= totall) {
            System.out.printf("You have %.2f leva left!", budget - totall);
        }else {
            System.out.printf("Not enough money! You need %.2f leva more!", totall - budget);
        }
    }
}

File: Programming Basics/JavaScript/Exam/Shopping.js
function solve(input) {
    let budget = Number(input.shift());
    let videoCards = Number(input.shift());
    let processors = Number(input.shift());
    let ram = Number(input.shift());
    let videoCardsPrice = videoCards * 250;
    let processorsPrice = processors * (0.35 * videoCardsPrice);
    let ramPrice = ram * (0.1 * videoCardsPrice);
    let totallCosts = videoCardsPrice + processorsPrice + ramPrice;
    if (videoCards > processors) {
        totallCosts *= 0.85;
    }
    if (totallCosts <= budget) {
        console.log(`You have ${(budget - totallCosts).toFixed(2)} leva left!`)
    } else {
        console.log(`Not enough money! You need ${(totallCosts - budget).toFixed(2)} leva more!`)
    }
}
solve([900, 2, 1, 3])